openquake/hazardlib/tests/gsim/chiou_youngs_2008_swiss_test.py

```diff
@@ -25,7 +25,7 @@ class ChiouYoungs2008SWISS01TestCase(BaseGSIMTestCase):
 
     def test_std_total(self):
         self.check('CY08Swiss/cy_2008_phis_ss_embeded.csv',
-                   max_discrep_percentage=1.50)
+                   max_discrep_percentage=2.00)
```

Comment:
This seems to imply that the modification of the reference vs30 means the tests are no longer passing at 1.5 % discrepancy. Shouldn't this be investigated?